android/src/main/java/com/reactnative/googlecast/api/RNGCSessionManager.java

```diff
@@ -73,6 +72,17 @@ public void run() {
     });
   }
 
+  @ReactMethod
+  public void endCurrentSession(final boolean stopCasting, final Promise promise) {
+    getReactApplicationContext().runOnUiQueueThread(new Runnable() {
+      @Override
+      public void run() {
+        getSessionManager().endCurrentSession(stopCasting);
```

Comment:
I saw there is already a method for ios, should I use this https://github.com/react-native-google-cast/react-native-google-cast/blob/v4/ios/RNGoogleCast/api/RNGCSessionManager.m#L66

and rename this method to match?

Reply:
Yeah, I guess it was copied from v3 but never properly set up. If you could hook it up that would be great!